VersionControlUI/styles/item/HighlightOverlay/insert

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE EnvisionVisualizationStyle>
```

Comment:
So all these styles, are very very similar. Please reduce them to the bare essentials, by using prototypes.